lib/Ix/DBIC/ResultSet.pm

```diff
@@ -906,6 +1005,16 @@ sub get_collection_lock ($self, $ctx) {
   };
 }
 
+=method ix_set($ctx, $arg)
+
+The method implemeents 'Foo/set'. It does some initial error checking and
```

Comment:
'implemeents' -> 'implements', unless this method is eeeeeeeevil

Reply:
👿